README.md

```diff
@@ -1,52 +1,68 @@
+  - Check skills about documentation (Code, Markdown, Wiki, etc.)
+  - Check skills about SCM ([Git](https://git-scm.com/))
+  - Check skills about scripting ([Python](https://pypi.org/), [Go](https://golang.org/), *sh, etc.)
+  - Check skills about containerization ([Docker](https://www.docker.com/))
+  - Check skills about packaging ([Helm Chart](https://helm.sh/))
+  - Check skills about orchestration ([Kubernetes](https://kubernetes.io/))
 
-_This is the list of the crystal clear test's goals._
 
-1. Check skills about code organization (Structure)
-2. Check skills about documentation (Code, Markdown, Wiki, etc.)
-3. Check skills about SCM ([Git](https://git-scm.com/))
-4. Check skills about scripting ([Python](https://pypi.org/), [Go](https://golang.org/), *sh, etc.)
-5. Check skills about containerization ([Docker](https://www.docker.com/))
-6. Check skills about packaging ([Helm Chart](https://helm.sh/))
-7. Check skills about orchestration ([Kubernetes](https://kubernetes.io/))
+## Getting Started
 
-### Bonus test goals
+In this repository we have two applications, one of which is in php `apps/hello`and the other in nodejs `apps/word`
 
-_This bonus are added to generate a discussion or to present a complete structure._
+#### 1. **Containerization of applications**
+Create and build the application image of  `apps/hello` and `apps/world`, each Dockerfile must be:
+   - [ ] Light as possible
+   - [ ] Embedded the notion of security
+   - [ ] Test configuration with docker-compose
 
-1. Check skills about a cloud provider (preference for [Google Cloud](https://cloud.google.com/) or [Amazon Web Services](https://aws.amazon.com/))
-2. Check skills about structure (VPC, Firewall, Nodes, Pools, etc.)
-3. Check skills about infrastructure as code ([Terraform](https://www.terraform.io/))
-4. Check skills about propositions for the next (Observability, Tracing, etc.)
+####  2. **Deployment of Kubernetes and ingress controller**
 
-## Test
+ - [ ] Install [Minikube](https://github.com/kubernetes/minikube)  or [Kind](https://kind.sigs.k8s.io/) with script (shell , go ,or python)
+ - [ ] Deploy [Traefik](https://docs.traefik.io/getting-started/install-traefik/)
+       as Ingress Controller with script (shell , go ,or python)
 
-### Kubernetes and ingress controller
+#### 3. **Package applications with helm**
 
-1. Install [Minikube](https://github.com/kubernetes/minikube) (script required)
-2. Deploy [Traefik](https://docs.traefik.io/getting-started/install-traefik/) as Ingress Controller (script required)
+ - [ ] Create the application charts of `apps/hello` and `apps/world`  
+ - [ ]  Add autoscalling notion
 
-### Containerize applications
+#### 4. **Deploy applications in kubernetes**
 
-1. Build the application images of `apps/hello` and `apps/world`
+ - [ ] Deploy the application `apps/hello` and `apps/world` in Install [Minikube](https://github.com/kubernetes/minikube) or [Kind](https://kind.sigs.k8s.io/) cluster using terraform
```

Comment:
Deploy the application `apps/hello` and `apps/world` on Minikube or kind cluster using Terraform (with minikube provider: https://registry.terraform.io/providers/scott-the-programmer/minikube/latest/docs/resources/cluster or kind provier: https://registry.terraform.io/providers/tehcyx/kind/latest/docs/resources/cluster)

Reply:
i don't want to provide the provider, the candidat must search and  implement

Reply:
You can specify to use Minikube/kind terraform provider